configure_log_agent.sh

```diff
@@ -163,8 +181,12 @@ echo PROJECT_DOMAIN_NAME  = "${PROJECT_DOMAIN_NAME}"
 echo DIMENSIONS    = "[ \"hostname:$HOSTNAME\"]"
 echo -e INPUT FILE\(S\) PATH\(S\) = "${FILES}"
 
-# Generate agent.conf file with specified values
-generate_config_file
+# Generate agent.conf file
+if [ ! "${FILES}" ]; then
```

Comment:
use `-z` switch (checks that string is empty) rather than `!` as $FILES does exist, it's just empty.

Reply:
Done.